spec/jobs/bookmark_reminder_notifications_spec.rb

```diff
@@ -59,4 +59,9 @@
       end
     end
   end
+
+  it 'will not send notification when topic is not available' do
```

Comment:
Minor but this description doesn't seem to match what the test does - it is checking that no error is raised, not whether a notification is created.